reference/docs-conceptual/components/vscode/How-To-Replicate-the-ISE-Experience-In-VSCode.md

```diff
@@ -21,161 +19,148 @@ to make the user experience a bit more familiar compared to the ISE.
+  To keep the focus in the editor when you execute with <kbd>F8</kbd>:
 
-> [!NOTE]
-> This setting is included in ["ISE Mode"](#ise-mode).
+  ```json
+  "powershell.integratedConsole.focusConsoleOnExecute": false
+  ```
 
-## No focus on console when executing
+  The default is `true` for accessibility purposes.
 
-To keep the focus in the editor when you execute with <kbd>F8</kbd>:
+  - Don't start integrated console on startup
 
-```json
-"powershell.integratedConsole.focusConsoleOnExecute": false
-```
+  To stop the integrated console on startup, set:
 
-> [!NOTE]
-> This setting is included in ["ISE Mode"](#ise-mode).
+  ```json
+  "powershell.integratedConsole.showOnStartup": false
+  ```
 
-The default is `true` for accessibility purposes.
+  > [!NOTE]
+  > The background PowerShell process will still start since that provides IntelliSense, script
+  > analysis, symbol navigation, etc. But the console won't be shown.
 
-## Don't start integrated console on startup
+- Assume files are PowerShell by default
 
-To stop the integrated console on startup, set:
+  To make new/untitled files, register as PowerShell by default:
 
-```json
-"powershell.integratedConsole.showOnStartup": false
-```
+  ```json
+  "files.defaultLanguage": "powershell",
+  ```
 
-> [!NOTE]
-> The background PowerShell process will still start since that provides IntelliSense, script analysis, symbol navigation, etc.
-But the console won't be shown.
-
-## Assume files are PowerShell by default
+- Color scheme
 
-To make new/untitled files, register as PowerShell by default:
+  There are a number of ISE themes available for VS Code to make the editor look much more like the
+  ISE.
 
-```json
-"files.defaultLanguage": "powershell",
-```
+  In the [Command Palette][] type `theme` to get `Preferences: Color Theme` and press
+  <kbd>Enter</kbd>. In the drop-down list, select `PowerShell ISE`.
 
-> [!NOTE]
-> This setting is included in ["ISE Mode"](#ise-mode).
+  You can set this theme in the settings with:
 
-## Color scheme
+  ```json
+  "workbench.colorTheme": "PowerShell ISE",
+  ```
 
-There are a number of ISE themes available for VSCode
-to make the editor look much more like the ISE.
+- PowerShell Command Explorer
 
-In the [Command Palette]
-type `theme` to get `Preferences: Color Theme` and press <kbd>Enter</kbd>.
-In the drop-down list, select `PowerShell ISE`.
+  Thanks to the work of [@corbob](https://github.com/corbob), the PowerShell extension has the
+  beginnings of its own command explorer.
 
-You can set this theme in the settings with:
+  In the [Command Palette][], enter `PowerShell Command Explorer` and press <kbd>Enter</kbd>.
 
-```json
-"workbench.colorTheme": "PowerShell ISE",
-```
+- Open in the ISE
 
-> [!NOTE]
-> This setting is included in ["ISE Mode"](#ise-mode).
+  If you end up wanting to open a file in the ISE anyway,
+  you can use <kbd>Shift</kbd>+<kbd>Alt</kbd>+<kbd>P</kbd>.
```

Comment:
This keybinding doesn't exist anymore... but it's still in the command pallet:
![image](https://user-images.githubusercontent.com/2644648/78834762-e9f50300-79a3-11ea-864b-7bacea4e347d.png)

Reply:
fixed